### PR TITLE
Update contributing md

### LINKED ISSUE #296: Update NestBot local set up documentation
**Is your feature request related to a problem? Please describe.**
The NestBot development is not covered well in the CONTRIBUTING.md

**Describe the solution you'd like**
Update docs with detailed step of how to set up NestBot development environment using ngrok. Include account registration and static domain set up steps.

### PR DESCRIPTION
**Pull request by ShashaankS**
Resolves #296

Added NestBot development Documentation to Contributing.md

- Added ngrok setup
- Added update for env variables
- Added setting up request url to slash commands

### CODE REVIEW

**Comment by ShashaankS** [comment]
@arkid15r I am a bit confused, do we have to add entire slack app setup also in the docs ?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -142,9 +148,55 @@ If you plan to fetch GitHub OWASP data locally, follow these additional steps:
...
+        NestBot:
+          addr: 8000
+          proto: http
+          hostname: <your-static-domain>
+     ```
+
+   - Now ngrok is all set, you access your local setup over internet, running the follwing command:
+
+     ```bash
+     ngrok start NestBot
```
Comment (arkid15r) [inline CONTRIBUTING.md]: @ShashaankS have you tested these set up instructions?

Comment (ShashaankS) [reply CONTRIBUTING.md:180]: Yes, I have tested it

Screenshot from 2024-12-31 00-31-04

This is my config file and it works well.

Comment (arkid15r) [reply CONTRIBUTING.md]: It's a bad idea to share your auth tokens.

What ngrok plan do you have?

Comment (ShashaankS) [reply CONTRIBUTING.md]: I got you, I won't share them.
Its the free one.

Comment (arkid15r) [reply CONTRIBUTING.md]: Okay, I'll check my config later and let you know.

**Review by arkid15r — APPROVED** [review]
LGTM 👍

**Pull request merged**